cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -126,7 +127,7 @@ func TestEvaluateExecute(t *testing.T) {
 			},
 
 			ExpectedOutputValidate: func(t *testing.T, output string, resultPath string) {
-				assert.Contains(t, output, `Evaluation score for "symflower/symbolic-execution" ("code-no-excess"): score=14, coverage-statement=10, files-executed=1, response-no-error=1, response-no-excess=1, response-with-code=1`)
+				assert.Regexp(t, `.*Evaluation score for "symflower/symbolic-execution" \("code-no-excess"\): score=14, coverage-statement=10, files-executed=1, processing-time=\d+, response-no-error=1, response-no-excess=1, response-with-code=1.*`, output)
```

Comment:
Since we are testing with a real execution, please make sure that procsessing time is > 0